Apps/GatewayApi/src/appsettings.Development.json

```diff
@@ -39,5 +39,10 @@
     "PhsaV2": {
         "TokenBaseUrl": "https://phsa-ccd-dev.azure-api.net/identity",
         "BaseUrl": "https://phsa-ccd-dev.azure-api.net/healthgateway"
+    },
+    "ChangeFeed": {
+        "Dependents": {
+            "Enabled": true
+        }
     }
```

Comment:
Shouldn't this be added to appsettings.json instead, and overridden here if needed?

Reply:
@sslaws and I discussed this, we'll add the flags to the common config in OCP to allow for explicitly setting this. However, it is not optional now, I'll make the code more tolerant when it is not configured.

Reply:
It looked like the code wouldn't fail it wasn't set, but would default to true.

Reply:
changed default to false